src/main/proto/event.proto

```diff
@@ -91,6 +91,16 @@ message QueryEventsRequest {
 
     /* Whether to keep the query running against incoming events once the Head of the Stream is reached */
     bool live_events = 3;
+
+    /* Indicates whether it is ok to query events from the local Axon Server node - the node the client is currently
+     * connected to. This means that the client will probably get stale events since all events my not be replicated to
+     * this node yet. Used when criteria for eventual consistency is loosen up. It will spread the load for querying
+     * events - not all requests will go to the leader of the cluster anymore.
+     * <p>
+     * If Axon Server SE is used, this property has no effect.
+     * </p>
+     */
+    bool use_local_store = 4;
```

Comment:
I think using the term ‘local’ can be confusing, since we’re in a client-server context. I suggest renaming to ‘allow_reading_from_follower’ or something along those lines.